src/Volume.ts

```diff
@@ -242,18 +287,19 @@ export default class Volume {
       this.pixel_size[2] = values[2];
     }
 
+    const sizez = this.imageInfo.vol_size_z || this.z;
     const physSizeMin = Math.min(this.pixel_size[0], this.pixel_size[1], this.pixel_size[2]);
-    const pixelsMax = Math.max(this.imageInfo.width, this.imageInfo.height, this.z);
+    const pixelsMax = Math.max(this.imageInfo.width, this.imageInfo.height, sizez);
     const sx = ((this.pixel_size[0] / physSizeMin) * this.imageInfo.width) / pixelsMax;
     const sy = ((this.pixel_size[1] / physSizeMin) * this.imageInfo.height) / pixelsMax;
-    const sz = ((this.pixel_size[2] / physSizeMin) * this.z) / pixelsMax;
+    const sz = ((this.pixel_size[2] / physSizeMin) * sizez) / pixelsMax;
```

Comment:
A comment below says that these variables, which are only ever used to set `this.scale`, are the same as `this.normalizedPhysicalSize`. Some `console.log`s show that `this.scale` and `this.normalizedPhysicalSize` are indeed always the same. `scale` is predominantly used in `RayMarchedAtlasVolume` and `Atlas2DSlice`, but `PathTracedVolume` exclusively uses `normalizedPhysicalSize`. Is there a future scenario where they will be different and it will be meaningful to use one or the other? Keeping just one could simplify this function a lot.

Reply:
`scale` can be removed if it is replaced by a unit test that confirms that this is equal